### PR TITLE
#37 ceil

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Closes: #37

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@Graur please take a look

File: src/main/eo/org/eolang/math/number.eo [hunk 1]
```diff
@@ -151,6 +153,27 @@
...
+        is-nan
+        is-int
+        eq.
+          value
+          positive-infinity
+        eq.
+          value
+          negative-infinity
+      value
+      []
```
Comment (Graur) [inline src/main/eo/org/eolang/math/number.eo:170]: @maxonfjvipon Why do we need this abstract object here? To improve readability?

Comment (maxonfjvipon) [reply src/main/eo/org/eolang/math/number.eo:170]: @Graur I'm not sure. Rather for optimization purposes (caching `as-int`). If you don't like it I can do it without an abstract object

Comment (Graur) [reply src/main/eo/org/eolang/math/number.eo:170]: @maxonfjvipon Just asking. I think we can keep it as is

**Review by Graur — APPROVED** [review]

**Comment by Graur** [comment]
@yegor256 Please, have a look

**Comment by yegor256** [comment]
@rultor merge

**Comment by Graur** [comment]
@maxonfjvipon Thanks!

**Comment by yegor256** [comment]
@maxonfjvipon thanks!

**Pull request merged**